data/assets/scene/solarsystem/sssb/pha.asset

```diff
@@ -2,7 +2,9 @@ local assetHelper = asset.require('util/asset_helper')
 local sharedSssb = asset.require('./sssb_shared')
 
 local filepath = sharedSssb.downloadSssbDatabaseFile(asset, 'pha', 'sssb_data_pha')
-local object = sharedSssb.createSssbGroupObject("sssb_data_pha.csv", filepath, { 0.75, 0.2, 0.2 })
+local object = sharedSssb.createSssbGroupObject('sssb_data_pha.csv', "Potentially Hazardous Asteroids", filepath, { 0.75, 0.2, 0.2 })
```

Comment:
```suggestion
local object = sharedSssb.createSssbGroupObject('sssb_data_pha.csv', "Potentially Hazardous Asteroid", filepath, { 0.75, 0.2, 0.2 })
```
just making plurality consistant

Reply:
Good catch, but actually I decided to do the opposite and make the other names plural as they should be. I had auto-generated the gui names based on their filenames, but now it's better.